compliance/oscal-assessment-results.yaml

```diff
@@ -202,107 +302,1265 @@ assessment-results:
+          uuid: 46256c29-c662-4f0c-a037-bf1c534dee59
+        - collected: 2024-08-06T02:58:06.749255782Z
+          description: |
+            [TEST]: fbe5855d-b4ea-4ff5-9f0d-5901d620577a - lula-validation-error
+          methods:
+            - TEST
+          relevant-evidence:
+            - description: |
+                Result: not-satisfied
+              remarks: |
+                Error getting Lula validation #fbe5855d-b4ea-4ff5-9f0d-5901d620577a: validation failed: [{/required file:///home/runner/work/uds-core/uds-core/compliance/validation#/required  missing properties: 'domain', 'provider' <nil>}]
+          uuid: af55317a-a3b8-42b9-8ba8-d859748635b5
+        - collected: 2024-08-06T02:58:06.85546841Z
+          description: |
+            [TEST]: 1761ac07-80dd-47d2-947e-09f67943b986 - all-pods-istio-injected
+          methods:
+            - TEST
+          relevant-evidence:
+            - description: |
+                Result: satisfied
+              remarks: |
+                validate.exempt_namespaces_msg: Exempted Namespaces: istio-admin-gateway, istio-passthrough-gateway, istio-system, istio-tenant-gateway, kube-system, uds-dev-stack, zarf
+                validate.msg: All pods have Istio sidecar proxy.
+          uuid: 435f54e2-3606-4250-9e16-79326844e82e
+        - collected: 2024-08-06T02:58:06.861988088Z
+          description: |
+            [TEST]: ca49ac97-487a-446a-a0b7-92b20e2c83cb - enforce-mtls-strict
+          methods:
+            - TEST
+          relevant-evidence:
+            - description: |
+                Result: satisfied
+              remarks: |
+                validate.msg: All PeerAuthentications have mtls mode set to STRICT.
+          uuid: dbc9b893-9847-4ffa-8a91-7642f18f9784
+        - collected: 2024-08-06T02:58:06.868755824Z
+          description: |
+            [TEST]: fd071676-6b92-4e1c-a4f0-4c8d2bd55aed - ingress-traffic-encrypted
+          methods:
+            - TEST
+          relevant-evidence:
+            - description: |
+                Result: satisfied
+              remarks: |
+                validate.msg: All gateways encrypt ingress traffic
+                validate.msg_exempt: Exempted Gateways: istio-passthrough-gateway/passthrough-gateway
+          uuid: ff67f994-802a-4668-a261-f0cbfb7982d5
+        - collected: 2024-08-06T02:58:06.868804496Z
+          description: |
+            [TEST]: 73434890-2751-4894-b7b2-7e583b4a8977 - lula-validation-error
+          methods:
+            - TEST
+          relevant-evidence:
+            - description: |
+                Result: not-satisfied
+              remarks: |
+                Error getting Lula validation #73434890-2751-4894-b7b2-7e583b4a8977: validation failed: [{/required file:///home/runner/work/uds-core/uds-core/compliance/validation#/required  missing properties: 'domain', 'provider' <nil>}]
```

Comment:
Is this expected? Just trying to make sure I understand where this is coming from.

Reply:
It is for now.

https://github.com/defenseunicorns/uds-core/blob/94766aa6bebb6464c30f5acc5d24a549e45c9609/src/istio/oscal-component.yaml#L365

Its a placeholder for FIPS. We have an open issue for how to determine if its the FIPs image for unicorn flavor of Istio.